src/lib/components/search/ArticleSearchResult.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import type { ArticleSearchReturnAttributes } from "$lib/search/searchTypes";
+  export let article: ArticleSearchReturnAttributes;
+</script>
+
+<li>
+  <a
+    href={"/news/" + article.slug}
+    class="border border-transparent focus:border-primary"
+    id=".search-result"
```

Comment:
What does this id do? And why this it start with a period? AFAIK id's are supposed to be globally unique.

Reply:
This is a part that enables keyboard navigation with the search results, used in `captureListItems()`
```ts
listItems = Array.from(formElement?.querySelectorAll("a")).filter(
  (a) => a.id === ".search-result",
);
```
I agree that this isn't idiomatic, just changed it in my latest commit